al2.pkr.hcl

```diff
@@ -1,18 +1,21 @@
 locals {
-  ami_name_al2 = "${var.ami_name_prefix_al2}-hvm-2.0.${var.ami_version}-x86_64-ebs"
+  ami_name_al2 = "kontain-ecs-${var.ami_name_prefix_al2}-hvm-2.0.${var.ami_version}-x86_64-ebs"
+  km_build = "../../../build"
+  target_tmp = "/tmp/"
 }
 
 source "amazon-ebs" "al2" {
   ami_name        = "${local.ami_name_al2}"
-  ami_description = "Amazon Linux AMI 2.0.${var.ami_version} x86_64 ECS HVM GP2"
-  instance_type   = "c5.large"
+  ami_description = "Kontain Amazon Linux AMI 2.0.${var.ami_version} x86_64 ECS HVM GP2"
+  instance_type   = "t2.micro"
```

Comment:
wondering if we should do t2.small or t2.medium, here because t2.micro is never used in an ECS setting because it is a very inferior VM type with bad networking etc.  so wondering if the AMI image might capture some characteristics that may not be a good starting point for a worker node.

Just something to think about.

Reply:
I changed it back to what Amazon used c5.large